packages/inter-protocol/src/auction/scheduler.js

```diff
@@ -147,18 +147,33 @@ export const makeScheduler = async (
    * @type {Schedule | undefined}
    */
   let liveSchedule;
-  /**
-   * Next should always be defined after initialization unless it's paused
-   *
-   * @type {Schedule | undefined}
-   */
-  let nextSchedule;
+
+  /** @returns {Promise<{now:Timestamp, nextSchedule: Schedule}>} */
+  const initializeNextSchedule = async () => {
+    return E.when(
+      // XXX manualTimer returns a bigint, not a timeRecord.
```

Comment:
exploration into burning this debt https://github.com/Agoric/agoric-sdk/pull/7348